source/developer/mobile-developer-setup.rst

```diff
@@ -117,10 +189,122 @@ Android (Device)
 
 8. The installed APK may not be opened automatically. You may need to manually open the Mattermost app on your device.
 
+Start Developing and Building
+------------------------------------
+
+In order to develop and build the Mattermost mobile apps you'll need to get a copy of the source code. Also, it is best if you fork our mattermost-mobile repo since it is most likely that you'll want to contribute back your work.
+
+1. Fork the mattermost-mobile repository on GitHub.
+2. Clone your fork locally:
+ - Open a terminal 
+ - Change to a directory you want to hold your local copy 
+ - Run ``git clone https://github.com/<username>/mattermost-mobile.git`` if you want to use HTTPS, or ``git clone git@github.com:<username>/mattermost-mobile.git`` if you want to use SSH
+
+Important: <username> refers to the username or organization in GitHub that forked the repository.
+
+3. Change the directory to ``mattermost-mobile``.
+ - ``cd mattermost-mobile``
+
+4. Run ``make pre-run`` in order to install all the dependencies.
+ - It is highly important that you run everything with the make commands and avoid using npm or yarn to install dependencies.
```

Comment:
Why is it important to do that?

Reply:
cause if you use npm or yarn you may be skipping some steps and the app won't build correctly